MvvmCross/ViewModels/MvxNotifyPropertyChanged.cs

```diff
@@ -141,6 +141,18 @@ void raiseChange()
             }
         }
 
+        [NotifyPropertyChangedInvocator]
+        protected virtual bool SetProperty<T>(ref T storage, T value, Action afterAction, [CallerMemberName] string propertyName = null)
+        {
+            if (!SetProperty(ref storage, value, propertyName))
+            {
+                return false;
+            }
+
+            afterAction?.Invoke();
```

Comment:
So this will only be executed when it is true. I'm not sure that makes it always usable. Why not make the afterAction, `Action<bool>` and always call it, with the result as parameter.

Reply:
I propose having both ideas at the same time. Initial idea was to get rid of if condition when you want to do something after set property (especially if it's only one line of code). 

```
[NotifyPropertyChangedInvocator]
protected virtual void SetProperty<T>(ref T storage, T value, Action<bool> action, [CallerMemberName] string propertyName = null)
{
    action?.Invoke(SetProperty(ref storage, value, propertyName));
}

[NotifyPropertyChangedInvocator]
protected virtual bool SetProperty<T>(ref T storage, T value, Action afterAction, [CallerMemberName] string propertyName = null)
{
    if (SetProperty(ref storage, value, propertyName))
    {
        afterAction?.Invoke();
        return true;
    }

    return false;
}
```


If you agree, I will edit Multi-binding part of data-binding documentation like this (to show two alternative usages).

**Multi-binding**

In Swiss binding, each binding can only reference a single ViewModel property path.

This meant that if a ViewModel had 2 properties like `FirstName` and `LastName`, then the main way to create a display of the ful name was to create a new ViewModel property - e.g.:

```c#
private string _firstName;
public string FirstName
{
    get => _firstName;
    set => SetProperty(ref _firstName, value, () => RaisePropertyChanged(() => FullName));
}

private string _lastName;
public string LastName
{
    get => _lastName;
    set => SetProperty(ref _lastName, value, (setPropertyResult) => 
    {
        if (setPropertyResult)
        {
            RaisePropertyChanged(() => FullName);
        }
    });
}

public string FullName => _firstName + " " + _lastName;
```

Reply:
I really like the option of having Action<bool> and Action. Are you able to commit your code and let's get this feature merged.